docker/dbt/Dockerfile

```diff
@@ -11,8 +11,7 @@ RUN apt-get update && \
     apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/
 RUN pip3 install --upgrade pip
 RUN mkdir ${DBT_DIR}
-# TODO: change with release version
-RUN pip install git+https://github.com/starburstdata/dbt-trino.git
+RUN pip install dbt-trino==1.3
```

Comment:
`RUN pip install dbt-trino~=1.3.*`

Reply:
Oops, too fast. This is indeed not working as expected.

It seems however that `RUN pip install dbt-trino==1.3.*` also works.

Reply:
`~=` will always install the latest version compatible with the pattern.
`==` won't update to the newest version if the older version which matches the pattern is already installed. 

So I think `~=` is slightly better here.